libbeat/docs/outputconfig.asciidoc

```diff
@@ -181,6 +181,34 @@ The index root name to write events to. The default is the Beat name.
 For example "{beatname_lc}" generates "[{beatname_lc}-]YYYY.MM.DD" indexes (for example,
 "{beatname_lc}-2015.04.26").
 
+===== template
+
+The http://www.elastic.co/guide/en/elasticsearch/reference/current/indices-templates.html[index
+template] to use for setting mappings in Elasticsearch. By default, template loading is
+enabled. If you disable this option, you must load the template by running the
+load script that is provided with {beatname_uc}. 
```

Comment:
AFAIK there isn't a load script, just some commands to execute on the command line. We can link to the instructions for loading the templates.